### DISCUSSION TITLE
How to programmatically display a message on a USB-connected camera?

### DISCUSSION THREAD

**Discussion by syenchuk** [gopro/labs#726] [Q&A]
Hello,

I am playing around with the [HTTP and Bluetooth APIs](https://gopro.github.io/OpenGoPro/), but so far, I wasn't been able to find a way to display a message on the camera.

I want to transfer photos to the computer, and show a message while the transfer is ongoing, and another one once the transfer is complete.

Is there an API or a hack for that? I didn't find anything in the Open Gopro specs. I was wondering if there was a hack through GoPro Labs.

Or is there a hidden way to execute a GoPro Labs script that would display a message using the HTTP or Bluetooth API?

**Comment by dnewman-gpsw** [comment]
The is no way yet, to use Open GoPro to send Labs commands, though it has been officially requested.  Labs is not aware of transfers to happening via the Linux subsystem as Labs runs deeper on the RTOS.  The best you can do it report when the USB cable is connected.
